app/public/javascripts/admin.js

```diff
@@ -99,11 +99,13 @@ function btnSetName(event) {
 
 /* stop people from signing up on the queue */
 function freeze_queue() {
+    queueFrozen = !queueFrozen;
```

Comment:
I don't exactly know if this change works on its own: what if someone else freezes the queue? Whether the queue is frozen should be tracked on server side, not client, and we'll need a getter just like the way we handle how the email notification button should look.